src/components/FoodIndicator.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import style from "../App.module.css";
+
+
+export const FoodIndicator = ({foodIndicator}) => {
```

Comment:
foodIndicator prop could be named something like `foodPercentage` or something that would indicate that we are using percents here. Not critical though